### ISSUE TITLE
change airodump-ng txt-file format

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#43]
**Reported by Ace on 5 Jan 2007 23:38 UTC**

It would be very nice if someone could change this:

  * the *.txt is now a binary file (not so bad, but can't edit it for testing)
  * if airodump can't get specific informations, it writes nothing to the file instead of a blank (" ")

e.g. :

```
BSSID, First time seen, Last time seen, channel, Speed, Privacy, Cipher, Authentication, Power, # beacons, # IV, LAN IP, ID-length, ESSID, Key
00:03:C9:B5:4C:29, 2007-01-05 12:02:17, 2007-01-05 12:02:21, 11,  48, OPN '''!,,,'''   6,       31,        0,   0.  0.  0.  0,   8,         ,
```

should be :

```
BSSID, First time seen, Last time seen, channel, Speed, Privacy, Cipher, Authentication, Power, # beacons, # IV, LAN IP, ID-length, ESSID, Key
00:03:C9:B5:4C:29, 2007-01-05 12:02:17, 2007-01-05 12:02:21, 11,  48, OPN ''', , ,'''   6,       31,        0,   0.  0.  0.  0,   8,         ,
```

link to forum post: http://tinyshell.be/aircrackng/forum/index.php?topic=982.0

**Comment by aircrack-ng** [comment]
**Modified by misterx on 5 Jan 2007 23:38 UTC**

**Comment by aircrack-ng** [comment]
**Comment by hirte on 5 Jan 2007 23:38 UTC**

(In #81) Fixed .txt output bug, used stderr instead of G.f_txt (Closes: #43).
